tools/lotus2/lotus2.xml

```diff
@@ -258,7 +284,8 @@ exit \$EXIT_VALUE
             <conditional name="amplicon_cond">
                 <param argument="-amplicon_type" type="select" label="Amplicon type">
                     <option value="LSU">LSU Large subunit (23S/28S)</option>
-                    <option value="SSU" selected="true">SSU small subunit (16S/18S)</option>
+                    <option value="SSU">SSU small subunit (16S/18S)</option>
+                    <option value="" style="color:gray" selected="true">(Default)</option>
```

Comment:
I'd put this option first.

Reply:
ok, but isn't it last currently?

Reply:
SSU (the current default) is in the middle because I think "LSU, SSU, ITS, ITS1, ITS2" was a good order, but the new "(Default)" should go first. I've added suggestions to implement that.
```suggestion
```